test/unit_test/arguments_boost_po_test.cc

```diff
@@ -0,0 +1,319 @@
+#define BOOST_TEST_DYN_LINK
+
+#include <boost/test/unit_test.hpp>
+#include <boost/test/test_tools.hpp>
+
+#include "test_common.h"
+
+#include "arguments_boost_po.h"
+#include "arguments_serializer_boost_po.h"
+
+#include <memory>
+#include <vector>
+
+using namespace VW;
+
+
+char** convert_to_command_args(char* command_line, int& argc, int max_args = 64) {
+  char** argv = new char*[max_args];
+  argc = 0;
+
+  char* current_arg = strtok(command_line, " ");
```

Comment:
nit:  prefer strtok_r to strtok

Reply:
I can't seem to be able to find the header that the _r version is defined in?